scripts/common_image

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/bash
```

Comment:
Please revert to the previous portable shebang everywhere.

Reply:
done